src/utils/query/wagmi.ts

```diff
@@ -128,25 +122,6 @@ const wagmiConfig_ = createConfig({
   },
 })
 
-const isSupportedChain = (chainId: number): chainId is (typeof chains)[number]['id'] =>
-  chains.some((c) => c.id === chainId)
-
-// hotfix for wagmi bug
-wagmiConfig_.subscribe(
-  ({ connections, current }) => (current ? connections.get(current)?.chainId : undefined),
-  (chainId_) => {
-    const chainId = chainId_ || chains[0].id
-    // If chain is not configured, then don't switch over to it.
-    const isChainConfigured = isSupportedChain(chainId)
-    if (!isChainConfigured) return
-
-    return wagmiConfig_.setState((x) => ({
-      ...x,
-      chainId: chainId ?? x.chainId,
-    }))
-  },
-)
-
```

Comment:
we still need these unless they went somewhere else, our logic means we're always defaulting to the 0th item in the chain array (without it it'll stay on the last connected supported chain)